src/Inventory/CreateList.js

```diff
@@ -35,7 +35,7 @@ class CreateList extends React.Component {
 			return;
 		}
 
-		Meteor.call('inventorylists.create', this.state.name, (err, newListId) => {
+		Meteor.call('inventorylists.create', this.state.name.trim(), (err, newListId) => {
```

Comment:
Ah nice, this is smart 😄

Reply:
Actually, instead of doing `.trim()` in the UI, why don't we just add this to the API so that it automatically trims all whitespaces?